tests/test_crypto.py

```diff
@@ -3389,6 +3398,14 @@ def test_load_crl_bad_data(self):
         """
         self.assertRaises(Error, load_crl, FILETYPE_PEM, b"hello, world")
 
+    def test_get_issuer(self):
+        """
+        Load a known CRL and inspect its issuer's common name.
```

Comment:
Please write test docstrings as what the expected behavior that you are testing, not about what you do: https://jml.io/pages/test-docstrings.html